CHANGES.md

```diff
@@ -2,6 +2,12 @@
 
 ### v0.6.1
 
+* Bugfix for `_create_group_metric_set()`. Fixes the list of metrics computed for regression
+  and adds a set of metrics for 'probability' problems
+* Updated 'Credit Card' notebook
+* Added some overlooked `MetricFrame` tests
+* Various documentation tweaks and enhancements
```

Comment:
would it be possible to start having PR numbers (or links with `:pr:number` directive) here?

Reply:
Done. Although I can't seem to figure out the short syntax. A bare `#<n>` doesn't work, nor does `:pr:<n>`